docs/code/utilization_map.py

```diff
@@ -0,0 +1,7 @@
+from bokeh.io import show
+from powersimdata import Scenario
+
```

Comment:
It seems I forgot to declare `scenario`. Can you add `scenario = Scenario("3287"). Sorry
`

Reply:
No problem - done